freqtrade/optimize/backtesting.py

```diff
@@ -1377,7 +1377,7 @@ def backtest_loop(
                 self.wallets.update()
 
             # 4. Create exit orders (if any)
-            if not trade.has_open_orders:
+            if trade.has_open_position:
```

Comment:
That's a pretty pointless comparison if you ask me.
It can either be gone completely (no comparison necessary) if the trade can't be closed before this point - or if we assume the trade state does change because the exit order fills - then it's closed - so we can use use `.is_open` for the comparison here.